com.civicrm.hrjobroles/js/src/job-roles/controllers/hr-job-roles-controller.js

```diff
@@ -126,10 +126,10 @@ define([
              * @param end
              * @param {function} error
              */
-            $scope.validateDates = function (start, end, error) {
+            $scope.validateDates = function (start, end, contract_start, contract_end, error) {
                 DateValidation.setErrorCallback(error);
 
-                DateValidation.validate(start, end);
+                DateValidation.validate(start, end, contract_start, contract_end);
```

Comment:
Use camelCase (`contractStart`) instead of snake_case (`contract_start`) for variable/parameters names